tests/transactional/disable_selinux.pm

```diff
@@ -0,0 +1,59 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Package: transactional-update
+# Summary: Enable SELinux on transactional server
+#
+# Maintainer: QA-C team <qa-c@suse.de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use transactional qw(process_reboot);
+use version_utils qw(is_leap_micro is_microos is_sle_micro is_public_cloud);
+use Utils::Systemd qw(systemctl);
+use Utils::Logging 'save_and_upload_log';
+use transactional qw(trup_call check_reboot_changes process_reboot);
+use publiccloud::utils qw(allow_openqa_port_selinux);
+use bootloader_setup 'replace_grub_cmdline_settings';
+
+
+sub check_disabled {
```

Comment:
There is already a method doing this somewhere else: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/transactional/enable_selinux.pm#L33

Instead of copy-pasting methods in different modules, I would extract them to some common place in `lib/`, for example in `https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/selinuxtest.pm`

Reply:
Ah, just seen that Ricardo already suggested this :)

Reply:
Can you pls check if the changes are ok?

Reply:
looks good, thnx!